include/inexor/vulkan-renderer/wrapper/command_buffer.hpp

```diff
@@ -1,19 +1,18 @@
 #pragma once
 
-#include "inexor/vulkan-renderer/wrapper/device.hpp"
+#include <vulkan/vulkan_core.h>
 
 #include <cstdint>
 #include <string>
 #include <vector>
 
 namespace inexor::vulkan_renderer::wrapper {
 
+class Device;
```

Comment:
I just see it here.
Personally i do this for forward declarations:
```c++
// forward declarations
namespace inexor::vulkan_renderer::wrapper {
class Device;
} // namespace inexor::vulkan_renderer::wrapper

namespace inexor::vulkan_renderer::wrapper {
class CommandBuffer {
// ....
}
} // namespace inexor::vulkan_renderer::wrapper
```

Reply:
Does `clang-format` not account for this?

Reply:
Its just a matter of style.
Not a format issue.